packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -29,9 +33,17 @@ class SafeConsoleLike implements ConsoleLike {
   }
 }
 
+/** 
+ * A class abstracting the execution of tasks in separate child processes
```

Comment:
I would even drop the "task" reference and phrase this slightly more generally. something like "Facilities to execute single or multiple external processes, with flexible forwarding of child process output"

Reply:
Thanks :) I adjusted the comment to your suggestion